ccdproc/image_collection.py

```diff
@@ -652,10 +663,17 @@ def _generator(self, return_type,
 
             file_name = path.basename(full_path)
 
+            if 'unit' not in ccd_kwargs:
```

Comment:
Tests are passing using

```
if 'unit' not in ccd_kwargs:
                ccd_kwargs['unit'] = 'adu'
```
However, after removing this if block code ``TestImageFileCollection.test_headers`` will fail because 
`fits_ccddata_reader` in
```
return_options = {'header': hdulist[0].header,
                              'hdu': hdulist[0],
                              'data': hdulist[0].data,
                              'ccd': fits_ccddata_reader(full_path,
                                                         **ccd_kwargs)}
```
will throw ``ValueError: a unit for CCDData must be specified``.